src/helpers/themeHelper.ts

```diff
@@ -0,0 +1,5 @@
+import { createTheme } from 'baseui';
+import { overrides, primitives } from '../utils';
+
+
+export const theme = createTheme(primitives, overrides);
```

Comment:
Just a silly thing,in this file there is an extra line break :eyes:
```suggestion
import { createTheme } from 'baseui';
import { overrides, primitives } from '../utils';

export const theme = createTheme(primitives, overrides);
```

Reply:
Thanks! Done.